### PR TITLE
Add username and fullname for formatting

### PR DESCRIPTION
**Pull request by ggurbet**
Added username and fullname to make them usable in the custom formatting for the posts.

### CODE REVIEW

**Comment by ggurbet** [comment]
I want to make use of the poster's username. Currently only the `display_name`, which is in the form "Firstname Lastname @username" format, is available.

**Review by eviltrout — APPROVED** [review]

**Comment by eviltrout** [comment]
Seems good to me!

File: lib/discourse_chat/provider/telegram/telegram_provider.rb [hunk 1]
```diff
@@ -72,6 +72,8 @@ def self.message_text(post)
         I18n.t(
             "chat_integration.provider.telegram.message",
             user: display_name,
+            username: post.user.username,
+            fullname: full_name,
```
Comment (ggurbet) [inline lib/discourse_chat/provider/telegram/telegram_provider.rb:76]: @eviltrout, the added `username` and `fullname` are not working as expected and when used, give an error: '_The following interpolation key(s) are invalid: "username, fullname"_'. What is the correct way to fix this?
Would adding
```
username = "#{post.user.username}"
fullname = "#{full_name}"
```
before I18n.t() fix the issue? I am new to this and need advice. My thought is they need to be stringified before being passed as `username` and `fullname`.

Comment (eviltrout) [reply lib/discourse_chat/provider/telegram/telegram_provider.rb:76]: You added them to the call but did not add them to the locale file. Maybe we should revert this?

Comment (ggurbet) [reply lib/discourse_chat/provider/telegram/telegram_provider.rb:76]: If adding the code above fixes the issue, then I can create a new pull request with them. Currently this PR should not cause any problems. What is your suggestion?

**Pull request merged**